packages/react-card/Spec.md

```diff
@@ -0,0 +1,432 @@
+
+If users have a scenario that requires them to use actionable components within a clickable Card then the bubbling of events is expected to be managed by the users themselves.
+
+Mouse interaction
+
+- OnMouseEnter / Hover
+
+  Cursor style stays at default, rest state stays persistent no visual feedback is applied to the card surface
+
+- Hover & onClick (Checkbox)
+
+  Hover state & pressed state applies to checkbox component
+
+- onClick
+
+  (Checkbox) Card and Checkbox are both in selected state, clicking again toggles components to unselected state
+
+#### Disabled
+
+![Visual interaction of Disabled cards using the mouse](./assets/disabled.png)
+
+1. Unselected disabled
+
+   - Label: Selectable Card
+   - Role: Group
+   - State: Disabled, unselected
+
+2. Selected disabled
+
+   - Label: Selectable Card
+   - Role: Group
+   - State: Disabled, selected
+
+### Keyboard
+
+### Interactive cards
+
+![Visual interaction of interactive cards using the keyboard](./assets/interactive-interaction-keyboard.png)
+
+1. Rest
+
+2. Keyboard interaction
+
+   EnterKey Perfoms action on card “Open file, Open link in browser etc...”
+
+#### Selectable cards
+
+![Visual interaction of selectable cards using the keyboard](./assets/selectable-interaction-keyboard.png)
+
+1. Rest Default state is ‘unselected’
+2. Focus Focus rectangle is shown on card
+3. Focus Use spacebar to toggle state (selected/unselected)
+
+### Non-interactive cards
+
+![Visual interaction of non-interactive cards using the keyboard](./assets/non-interactive-interaction-keyboard.png)
+
+Keyboard interaction
+
+1. Focus
+
+   Initial focus is set on card, to access items within the card use EnterKey or Spacebar
+
+2. Focus-Inside
+
+   Focus moves to first actionable item in the card. Use ArrowKeys or TabKey to move focus to child elements within the card. To exit card use EscKey, to move focus back to card [parent]
+
+3. Checkbox-Focus
+
+   Depending on your tab order of items within the card, the number of keystrokes will vary. Once focus is set on the checkbox by default it will be set to unselected
+
+4. SpaceBar or EnterKey, while focused on the Checkbox, will toggle both Card and Checkbox to selected state. Keying again will toggle to unselected
+
+## Accessibility
+
+> ⚠️ Note: All other accessibility information, not covered in this section, is provided throughout the spec.
+
+Each card is of `role="group"`.
+
+### Narrator guidance
+
+### Interactive cards
+
+![A card with the number 1 over it](./assets/interactive-narrator.png)
+
+1. Label: App Card
+2. Role: Group
+3. State: Focus
+4. Helper text: Use enterKey to perform action
+
+#### Selectable cards
+
+![Visual interaction of keyboard usage with selectable cards](./assets/selectable-narrator.png)
+
+1 Focus + unselected
```

Comment:
It would be great to have some DOM output in the spec somewhere to make these points clearer

Reply:
Let me know if the DOM I added above for each Card component is enough or if I should also put it here.